### ISSUE TITLE
Improve contrast for cluster counts

### ISSUE THREAD

**Issue by creativecoder** [Automattic/automatticians-map#14]
> I wonder if using black text + white bg + blue border instead of just blue+white would work better keeping WCAG AA(A) contrast ratio in mind?
> 
> See how these colours fail for small text: https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=3499CD
> 
> Something more similar to this symbol? :-)
![Automattic logo](https://user-images.githubusercontent.com/87168/39464923-fe766a52-4d28-11e8-9a2b-04ac1f612ae3.png)
> 
> Note that better contrast would also improve the mobile experience.

**Comment by creativecoder** [comment]
Current with #3499cd background
<img width="82" alt="screen shot 2018-05-02 at 11 34 28" src="https://user-images.githubusercontent.com/1699996/39536978-ca94c8c0-4dfd-11e8-89e4-6b4c44325b73.png">

Background with same hue but darkened to #2a7eab for 1:4.5 contrast ratio
<img width="91" alt="screen shot 2018-05-02 at 11 34 44" src="https://user-images.githubusercontent.com/1699996/39537057-03ba1178-4dfe-11e8-8a46-07d2e93952ee.png">

Using #3499cd border with white background and black text
<img width="81" alt="screen shot 2018-05-02 at 11 37 03" src="https://user-images.githubusercontent.com/1699996/39537068-0b49fb10-4dfe-11e8-95f3-1ff9e52d5caf.png">

@evilluendas Any thoughts or suggestions for increasing contrast ratio?

**Comment by evilluendas** [comment]
I'd probably go with the first option (`#2a7eab`), but setting the opacity of the tag to 1 (fully opaque instead of translucent).

The second option adds a little bit too much noise, in my opinion, although it could work with just a subtle shadow:

![image](https://user-images.githubusercontent.com/820374/39537402-c957e1ae-4e39-11e8-9c8e-5abce368da83.png)

**Comment by creativecoder** [comment]
@evilluendas Thanks for the quick feedback! I've updated https://automattic.github.io/automatticians-map/leaflet/ to use option 1 (may need to clear browser cache to get latest css). Let me know if you'd like to adjust any further.
